cmd/create/cluster/cmd.go

```diff
@@ -1502,18 +1496,17 @@ func run(cmd *cobra.Command, _ []string) {
 	}
 
 	// Cluster-wide proxy configuration
-	enableProxy := args.enableProxy
+	enableProxy := false
 	httpProxy := args.httpProxy
 	httpsProxy := args.httpsProxy
 	additionalTrustBundleFile := args.additionalTrustBundleFile
-	if httpProxy != "" || httpsProxy != "" || additionalTrustBundleFile != "" {
-		enableProxy = true
-	}
```

Comment:
Why remove this? If the user sets `http-proxy` we can safely assume that they want to enable proxy and so we should set it to `true`, then skip the prompt below

Reply:
The user is not able to set any proxy value if enable-proxy is false. enable-proxy is relevant only for interactive mode. You can also check out the branch and try it, you will see what I mean.